core/modules/wiki.js

```diff
@@ -101,7 +101,7 @@ exports.deleteTextReference = function(textRef,currTiddlerTitle) {
 exports.addEventListener = function(type,listener) {
 	this.eventListeners = this.eventListeners || {};
 	this.eventListeners[type] = this.eventListeners[type]  || [];
-	this.eventListeners[type].push(listener);
+	this.eventListeners[type].push(listener);	
```

Comment:
what should these spaces be good for?

Reply:
They are not good at all, but deletion of whitespace is unrelated to this PR, but their removal would make `git blame` dirty.

Reply:
ah. ok. ...